### DISCUSSION TITLE
Job Announcement : gui2de RAships and Internships

### DISCUSSION THREAD

**Discussion by BeaLeydier** [gui2de/ppol768-spring23#325] [Announcements]
Hi @gui2de/ppol768-spring23 

gui2de is hiring RAs and summer interns! Most of them will be on project I work with Pr Andrew Zeitlin on : we have a government-facing large-scale education project in Rwanda, as well as two RCTs implemented by the IRC in Kenya and Uganda with microentrepreneurship support programs including a mentorship component. In both projects, we are looking for both RAs to work during the school year and summer interns to travel over the summer, so ideally first year master students (or junior undergrads). 

The postings are up in HoyaWorks through which you can apply directly. Please apply if you are interested, and forward to anyone who could be interested!

ReBuild (refugee microentrepreneurs in Kenya and Uganda) : https://seo.georgetown.edu/jobxJobdetailPrint.aspx?JobId=30102&win=True

STARS (education in Rwanda) : https://seo.georgetown.edu/jobxJobdetailPrint.aspx?JobId=30101&win=True

**Comment by BeaLeydier** [comment]
Note : you are welcome to apply to both if you are interested in both/either.

**Comment by BeaLeydier** [comment]
One more opportunity! Also a summer internship with gui2de, but this one in Bangladesh, on EV batteries recycling!

https://seo.georgetown.edu/jobxJobdetailPrint.aspx?JobId=30139&win=True

**Also: all applications will be open until April 10th, so make sure you put in your applications before then.**

**Comment by sumitshinde-84** [comment]
Hello,
My name is Sumit Shinde, and I am a passionate and hardworking full-stack developer. I have recently completed my education and am actively seeking job opportunities to kickstart my career. While I don't have any professional experience yet, I have gained valuable skills and knowledge through my education and by contributing to open-source projects.
